.github/markdown-linter-config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "ignorePatterns": [
+        {
+            "pattern": "https://crypto.stackexchange.com/questions/54852/what-happens-if-a-sha-256-input-is-too-long-longer-than-512-bits"
```

Comment:
👍 dope! In a future PR I can add the link from https://github.com/celestiaorg/celestia-app/issues/2475

Reply:
I think it works now: https://github.com/celestiaorg/celestia-app/actions/runs/6238291658/job/16933808665?pr=2539#step:4:99